.github/workflows/validate.yml

```diff
@@ -55,3 +55,49 @@ jobs:
         run: |
           pnpm cypress:verify
           pnpm cypress
+  test-mac:
+    name: Test on Mac
+    runs-on: macos-latest
```

Comment:
This can be collapsed with the above [using a matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs).

```suggestion
    runs-on: [ubuntu-latest, macos-latest]
```

Reply:
Requires a slightly more complicated syntax, but that seems to be working

Reply:
Actually no it isn't

Reply:
I think the failure has something to do with the caches. I cleaned all caches and re-ran all jobs.

Reply:
Looks like it's running both Cypresses now